CoreLayersView/src/au/gov/asd/tac/constellation/views/layers/shortcut/LayersViewShortcuts.java

```diff
@@ -52,74 +54,87 @@
 @NbBundle.Messages("CTL_LayersViewShortcuts=Layers View: Shortcuts")
 public class LayersViewShortcuts extends AbstractAction {
 
+    private void triggerLayerSelection(final int layerId) {
+        var vxCollection = LayersViewController.getDefault().getVxQueryCollection();
+        var txCollection = LayersViewController.getDefault().getTxQueryCollection();
```

Comment:
is there an alternative to declaring `var`?

Reply:
Forgot to remove this var when using it to test my thinking 👍